free-3.md

```diff
@@ -57,12 +67,13 @@ making language design decisions), as opposed to "systems" aspects of the
 project (e.g., implementing a complicated semantics that doesn't require a lot
 of language design)?_
 
+I imagine most of the work would be on the "systems" aspects, as the language is pretty simplistic in terms of the syntax. I'm sure there are ways in which we could emphasize language design decisions more, such as taking in arguments in an arbitrary order and recognizing what they are. Because there is some degree of natural language embedded in it, building some form of parsing might be also a good amount of work (I consider this parsing to be part of the language aspect, but I'm not sure if others do).
 
 ### Scope
 _How big an idea is this? How ambitious is this project?_
-
+This idea is very manageable compared to the other ones I've written about, and not very ambitious. Perhaps this project could be extended to handle class registration as well, which would increase the scope significantly since the implementer would have to handle some back end interactions with Portal.
 
```

Comment:
It's an interesting design challenge. I think a lot of the complexity would come from making a textual interface that gave the user enough freedom to schedule events with awareness of their current schedule. It sounds like a fun problem to tackle!